strawberryfields/decompositions.py

```diff
@@ -77,29 +77,29 @@ def takagi(N, tol=13):
     return rl, U
 
 
-def graph_embed(mat, max_mean_photon=1.0, make_traceless=True, tol=6):
+def graph_embed(mat, max_mean_photon=1.0, make_traceless=True, tol=1e-6):
     r""" Given an symmetric adjacency matrix (that can be complex in general),
-    it returns the squeezing parameter and interferometer necessary for
+    it returns the squeezing parameters and interferometer necessary for
```

Comment:
Just below: we should change "for implementing it in GBS" to something more generic like "for creating the Gaussian state whose off-diagonal parts are proportional to that matrix"